server/proto/get5-web-go.proto

```diff
@@ -319,6 +321,83 @@ message MatchEventDemoUpload {
   // binary...
 }
 
+message Player {
+  string name = 1;
+  int32 id = 2;
+  string steamid = 3;
+  string side = 4;
+}
+
+message Position {
+  int32 x = 1;
+  int32 y = 2;
+  int32 z = 3;
+}
+
+message MatchEventPlayerKill {
+  int32 matchid = 1;
+  Player Attacker = 2;
+  Position AttackerPosition = 3;
+  Player Victim = 4;
+  Position VictimPosition = 5;
+  string weapon = 6;
+  bool headhot = 7;
```

Comment:
Not sure if misspelling, but just a heads up! 😄

Reply:
Thanks for reviewing, that was my typo XD